src/main/kotlin/org/rust/lang/core/psi/ext/RsMod.kt

```diff
@@ -33,6 +33,8 @@ interface RsMod : RsQualifiedNamedElement, RsItemsOwner {
 
     val isCrateRoot: Boolean
 
+    val isPublic: Boolean
```

Comment:
Hm, perhaps we should split `VisibilityOwner` into `Visible` and `PubOwner: Visible`?